iothub/service/src/Utilities/ExceptionTrace.cs

```
@@ -53,12 +55,12 @@ public ArgumentNullException ArgumentNull(string paramName, string message)
 
         public ArgumentException ArgumentNullOrEmpty(string paramName)
```

Can we remove this in favor of `Arguments.AssertNotNullOrWhiteSpace()`?

---

Arguments.AssertNotNullOrWhiteSpace() throws an exception instead of returning it. We need to return the exception in this method.

---

Consider rewriting the caller to use that so we can get rid of this.

---

This method is not being used anywhere, removed it in a recent change.